resources/scponlyc_user.rb

```diff
@@ -0,0 +1,55 @@
+# To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+
+resource_name :scponlyc_user
```

Comment:
I think ``scponly_user`` might be a better name for this.

Reply:
I was thinking I could name it that and create another scponly_user without chroot incase we have a use for that later on? or I could define an attribute `:chroot` that defaulted true?

Reply:
I think adding a parameter which sets that is the better way to go.